{"name": "test_rerun", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2\n  +1\n  -2", "trace": "def test_rerun():\n>       assert 1 == 2\nE       assert 1 == 2\nE         +1\nE         -2\n\ntest_rerun.py:2: AssertionError"}, "start": 1631177778192, "stop": 1631177778192, "uuid": "76278367-bd3a-43f0-a266-36631c9a3d81", "historyId": "47c35f179f4b9f2c17e761a436dbbee0", "testCaseId": "558bc7562212911663d26c2ca57bd815", "fullName": "test_rerun#test_rerun", "labels": [{"name": "suite", "value": "test_rerun"}, {"name": "host", "value": "DESKTOP-GRKQKVO"}, {"name": "thread", "value": "44896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_rerun"}]}